demos/TransferLearning/TransferLearningDemo.py

```diff
@@ -0,0 +1,481 @@
+import warnings
+warnings.simplefilter('ignore')
+
+import ngraph_bridge
+ngraph_bridge.enable()
+#ngraph_bridge.set_backend('PLAIDML')
```

Comment:
Did you mean to leave this commented out?

Reply:
Intended to delete that line.

Reply:
There're some other commented bits later on... might need more cleanup?